proto/nexus/exported/v1beta1/types.proto

```diff
@@ -52,11 +54,22 @@ message TransferFee {
   ];
 }
 
-message Asset {
-  string denom = 1;
-  bytes min_amount = 2 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+message FeeInfo {
```

Comment:
Preferences on naming between `FeeInfo` vs `Fee` vs `AssetFee`?

Reply:
I think the current name is fine, just don't call the commands 'register-asset-fee-info'